python/hidet/drivers/build_module.py

```diff
@@ -118,20 +120,39 @@ def build_ir_module(ir_module: IRModule, output_dir: str, *, target: str, output
         if target.name == 'cpu' and 'arch' in target.attrs:
             hidet.option.cpu.arch(target.attrs['arch'])
         with PassContext(instruments=instruments):
-            ir_module = lower(ir_module)
+            if isinstance(ir_module, Sequence):
+                for i in range(len(ir_module)):
+                    ir_module[i] = lower(ir_module[i])
+            else:
+                ir_module = lower(ir_module)
 
     # code generation
     codegen(ir_module, src_out_path=src_path, target=target)
 
+    include_d = []
+    linking_d = []
+    linking_l = []
+    object_f = []
```

Comment:
I'd proposed to use dirs. libs, files instead of one letter

Reply:
thanks, name changed to dirs/lib/files